### PR TITLE
Setting Nginx docker to support several user/passwords in Kibana

### PR DESCRIPTION
**Pull request by toniMR**
Hi team,

In order to set users credentials, before " docker-compose up -d " run (a or b):
a) export NGINX_CREDENTIALS="user1:pass1;user2:pass2;"

b) Set NGINX_CREDENTIALS in docker-compose.yml:
   NGINX_CREDENTIALS=user1:pass1;user2:pass2;

### CODE REVIEW

File: nginx/config/entrypoint.sh [hunk 1]
```diff
@@ -12,15 +12,27 @@ else
...
+    do
+      IFS=':' read -r -a credentials <<< "${users[index]}"
+      if [ $index -eq 0 ]; then
+        echo ${credentials[1]}|htpasswd -i -c /etc/nginx/conf.d/kibana.htpasswd ${credentials[0]} >/dev/null
+      else
+        echo ${credentials[1]}|htpasswd -i /etc/nginx/conf.d/kibana.htpasswd  ${credentials[0]} >/dev/null
+      fi
+    done
+  else
+    echo $NGINX_PWD|htpasswd -i -c /etc/nginx/conf.d/kibana.htpasswd $NGINX_NAME >/dev/null
```
Comment (jesuslinares) [inline nginx/config/entrypoint.sh]: Where is declared $NGINX_PWD and $NGINX_NAME?

File: nginx/config/entrypoint.sh [hunk 2]
```diff
@@ -12,15 +12,27 @@ else
   echo "SSL certificates already present"
 fi
 
-# Configuring default credentiales.
+# Setting users credentials.
 if [ ! -f /etc/nginx/conf.d/kibana.htpasswd ]; then
-  echo "Setting Nginx credentials"
-  echo $NGINX_PWD|htpasswd -i -c /etc/nginx/conf.d/kibana.htpasswd $NGINX_NAME >/dev/null
+  echo "Setting users credentials"
+  if [ ! -z "$NGINX_CREDENTIALS" ]; then
```
Comment (jesuslinares) [inline nginx/config/entrypoint.sh]: Add a comment explaining the expected format of this variable.

File: docker-compose.yml [hunk 3]
```diff
@@ -57,6 +57,7 @@ services:
     restart: always
     environment:
       - NGINX_PORT=443
+      - NGINX_CREDENTIALS
```
Comment (jesuslinares) [inline docker-compose.yml]: Add default credentials.
NGINX_CREDENTIALS="foo:bar;"

File: nginx/config/entrypoint.sh (hunk 1 above)
Comment (toniMR) [reply nginx/config/entrypoint.sh]: In wazuh-docker/nginx/Dockerfile.

**Pull request merged**